src/components/service-provider-result.js

```diff
@@ -13,50 +13,45 @@ import ServiceClassification from './service-classification';
 export default class ServiceProviderResult extends Component {
   static propTypes = {
     index: PropTypes.number,
-    provider: PropTypes.object.isRequired,
     userLocation: PropTypes.object,
+    mappedProvider: PropTypes.object,
   };
 
   render() {
     const {
       index,
-      provider,
-      provider: {
-        FSD_ID: fsdId,
-        PROVIDER_CLASSIFICATION: classification,
-        ORGANISATION_PURPOSE: purpose,
-      },
       userLocation,
+      mappedProvider,
     } = this.props;
 
     return (
       <Card result-index={index} className="service">
         <CardContent>
-          {process.env.REACT_APP_DISPLAY_INDEX && (<div>{index+1}</div>) }
-          {process.env.REACT_APP_DISPLAY_INDEX && (<div>{provider.rank}</div>) }
```

Comment:
This is just for debugging purposes - if it's useful I could put `rank` through, but I'm not sure it is